code/game/objects/items/weapons/implants/implant_death_alarm.dm

```diff
@@ -68,4 +68,42 @@
 	if(..())
 		processing_objects.Remove(src)
 		return 1
-	return 0
\ No newline at end of file
+	return 0
+
+
+
+/obj/item/weapon/implant/death_alarm/dust
+	name = "duster implant"
+	desc = "An alarm which monitors host vital signs, transmitting a radio message and dusting the corpse on death."
+
+/obj/item/weapon/implant/death_alarm/dust/activate(var/cause)
```

Comment:
Commit I just did rewrote dust alarm implants to be more based on explosive implants, and remove the duplicate death alarm functionality entirely.

See what you think of it now.